packages/FlexLoader/src/Flex/FlexPathsFactory.php

```diff
@@ -10,7 +10,7 @@ final class FlexPathsFactory
     /**
      * @return string[]
      */
-    public function createServicePaths(string $projectDir, string $environment): array
+    public function createServicePaths(string $projectDir, string $environment, array $extraServicePaths = []): array
```

Comment:
Why default value for `$extraServicePaths`? Is it optional parameter anywhere in code?

Reply:
yes it is optional in:
` public function loadConfigs(ContainerBuilder $containerBuilder, LoaderInterface $loader, array $extraServicePaths = [])`
from which it's called so I made it the same, but it doesn't have to be as from call context it's always passed anyway.

Reply:
I see. Please make it required here, as it's part of internal api and that should be as strict as possible.

Reply:
Done.